runtime/src/lib.rs

```diff
@@ -200,11 +200,11 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
     let req_packet = Packet::get_from_mbox(drivers)?;
     let cmd_bytes = req_packet.as_bytes()?;
 
-    cprintln!(
-        "[rt] Received command=0x{:x}, len={}",
-        req_packet.cmd,
-        req_packet.payload().len()
-    );
+    // cprintln!(
+    //     "[rt] Received command=0x{:x}, len={}",
+    //     req_packet.cmd,
+    //     req_packet.payload().len()
+    // );
```

Comment:
is that commented out just to speed up the FPGA test?

Reply:
Oh, yes. I meant to put that back in :)